src/components/Forms/SubscribeForm.js

```diff
@@ -2,8 +2,12 @@ import React from 'react'
 import { Formik, Field, Form, ErrorMessage } from 'formik'
 import * as Yup from 'yup'
 import axios from 'axios'
+import tw from 'tailwind.macro'
 import { css } from '@emotion/core'
-import theme from '../../../config/theme'
+
+import colors from '../../lib/colors'
```

Comment:
how can we create alias for the folders like `lib` and `components`? 
Probably this may be useful? https://www.gatsbyjs.org/packages/gatsby-plugin-alias-imports/